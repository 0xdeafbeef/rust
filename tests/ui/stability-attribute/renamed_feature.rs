#![feature(try_trait)] //~ ERROR unknown feature `try_trait`

fn main() {}
