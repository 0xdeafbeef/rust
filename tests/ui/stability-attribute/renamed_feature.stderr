error[E0635]: unknown feature `try_trait`
  --> $DIR/renamed_feature.rs:1:12
   |
LL | #![feature(try_trait)]
   |            ^^^^^^^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0635`.
